### PR TITLE
Download image

### PR DESCRIPTION
**Pull request by alexandershpak**

### CODE REVIEW

File: src/public/js/build/base.js [hunk 1]
```diff
@@ -50,13 +50,50 @@ $(document).ready(function () {
...
         });
 
-        setSelectedTrackProperties();
-        getElementInstances(selectedTrackId);
-      });
+                setSelectedTrackProperties();
+                getElementInstances(selectedTrackId);
+            });
+
+            $('#import-microservice-modal-yes-button').click(function () {
```
Comment (alexandershpak) [inline src/public/js/build/base.js]: del

File: src/public/js/build/base.js [hunk 2]
```diff
@@ -50,13 +50,50 @@ $(document).ready(function () {
...
+                }
+            });
+        });
     });
-  });
 });
 
+const getElementInstanceImageSnapshot = function(accTok, elementId){
+   return $.ajax({
+        type: "POST",
```
Comment (alexandershpak) [inline src/public/js/build/base.js]: del

File: src/public/js/build/build-interface.js [hunk 3]
```diff
@@ -1702,4 +1716,89 @@ function clickRebuild() {
         });
     timer = setInterval(rebuildTimer1, 1000, elementId);
     selectedElement.data('timer1', timer);
+}
+
+function importMicroservice(json) {
+    $.ajax({
+        type: "POST",
```
Comment (alexandershpak) [inline src/public/js/build/build-interface.js]: del

File: src/public/js/build/build-interface.js [hunk 4]
```diff
@@ -1702,4 +1716,89 @@ function clickRebuild() {
...
+    var elementId = instanceId || selectedElement.attr('data-element-instance-id');
+
+    let initFetch = { method: 'POST',
+        cache: 'default' };
+
+    let urlReq = fogControllerUrl + ioAuthoring.urls.elementInstanceImageSnapshot + '?elementId=' + elementId + '&t=' + accTok;
+    let myRequest = new Request(urlReq, initFetch);
+    fetch(myRequest)
+        .then(function(response) {
+            console.log(response.type);
```
Comment (alexandershpak) [inline src/public/js/build/build-interface.js]: del log

File: src/public/js/build/build-interface.js [hunk 5]
```diff
@@ -1702,4 +1716,89 @@ function clickRebuild() {
...
+        type: "POST",
+        url: fogControllerUrl + ioAuthoring.urls.elementInstanceImageSnapshotStatus,
+        dataType: "json",
+        data: {
+            t: accTok,
+            elementId: elementId
+        },
+        success: function (result) {
+
+            if (result.status == "ok") {
```
Comment (alexandershpak) [inline src/public/js/build/build-interface.js]: add log about success

Comment (elukashick) [reply src/public/js/build/build-interface.js]: better to use logger.error

**Pull request merged**